docs/excel/custom-functions-overview.md

```diff
@@ -137,8 +137,8 @@ The following table lists the properties that are typically present in the JSON
 | `name` | Name of the function that the end user sees in Excel. In Excel, this function name will be prefixed by the custom functions namespace that's specified in the [XML manifest file](#manifest-file). |
 | `helpUrl`	| URL for the page that is shown when a user requests help. |
 | `description`	| Describes what the function does. This value appears as a tooltip when the function is the selected item in the autocomplete menu within Excel. |
-| `result` 	| Object that defines the type of information that is returned by the function. The value of the `type` child property can be **string**, **number**, or **boolean**. The value of the `dimensionality` child property can be **scalar** or **matrix** (a two-dimensional array of values of the specified `type`). |
-| `parameters` | Array that defines the input parameters for the function. The `name` and `description` child properties appear in the Excel intelliSense. The value of the `type` child property can be **string**, **number**, or **boolean**. The value of the `dimensionality` child property can be **scalar** or **matrix** (a two-dimensional array of values of the specified `type`). |
+| `result` 	| Object that defines the type of information that is returned by the function. The value of the `type` child property must be **string**, **number**, **boolean**, or **any**, which allows any of these three. **Any** is the default option if none is selected. The value of the `dimensionality` child property can be **scalar** or **matrix** (a two-dimensional array of values of the specified `type`). |
```

Comment:
Instead of duplicating this information about child properties here, I'd suggest the following:
- Remove all but the first sentence of this description. 
- After the first sentence, add a pointer to the detailed info about this object -- something like: For detailed information about this object, see `[result](custom-functions-json.md#result)`.